src/core/core.datasetController.js

```diff
@@ -399,7 +392,7 @@ export default class DatasetController {
 			me.getDataset(),
 		], {
 			merger(key, target, source) {
-				if (key !== 'data') {
+				if (key !== 'data' && key.charAt(0) !== '_') {
```

Comment:
this line could probably use a comment explaining

Reply:
Added a comment. Without it, when the plugin stored the original data onto `dataset._data` it got cloned and slowed everything down. Another option might be to provide some standard storage mechanism for plugins, but this seemed super simple and easy to reason about.